web/src/components/njwds-extended/cta/SingleCtaLink.tsx

```diff
@@ -33,6 +35,11 @@ export const SingleCtaLink = (props: Props): ReactElement => {
           }}
         >
           {props?.text || Config.taskDefaults.defaultCallToActionText}
+          {props.iconName && (
+            <Icon className="text-green usa-icon--size-3 margin-left-1 padding-bottom-05">
+              {props.iconName}
```

Comment:
Not something you chose, but I think it's super odd that we specify the icon type by passing a string as a child. I kinda hate this.

Reply:
Totally agree and I think it's a great opportunity for a refactor ticket!

Reply:
Made a [ticket](https://www.pivotaltracker.com/story/show/188448975). Marking resolved.